services/order.py

```diff
@@ -0,0 +1,35 @@
+from datetime import datetime
+from db.models import Order, User, Ticket
+from django.db.models.query import QuerySet
+from django.db import transaction
+from services.movie_session import get_movie_session_by_id
+
+
+@transaction.atomic
+def create_order(tickets: list[dict], username: str, date: str = None) -> None:
+    user = User.objects.get(username=username)
```

Comment:
Consider handling the case where the `User` object with the given `username` does not exist. This will prevent a `DoesNotExist` exception from being raised.